services/user/Registry/plugin/src/lib.rs

```diff
@@ -0,0 +1,217 @@
+#[allow(warnings)]
+mod bindings;
+
+use bindings::exports::registry::plugin::consumer::Guest as Consumer;
+use bindings::exports::registry::plugin::developer::Guest as Developer;
+use bindings::host::common::{server as Server, types as CommonTypes};
+use bindings::registry::plugin::types::{
+    AccountId, AppMetadata, AppStatus, ExtraMetadata, FullAppMetadata,
+};
+use bindings::transact::plugin::intf as Transact;
+use chrono::DateTime;
+use psibase::fracpack::Pack;
+use registry as RegistryService;
+use registry::service::AppMetadata as ServiceAppMetadata;
+use registry::service::AppStatusU32 as ServiceAppStatusU32;
+use registry::service::TagRecord;
+
+use serde::Deserialize;
+
+mod errors;
+use errors::ErrorType::*;
+
+struct RegistryPlugin;
+
+trait TryParseGqlResponse: Sized {
+    fn from_gql(s: String) -> Result<Self, CommonTypes::Error>;
+}
+
+#[derive(Deserialize, Debug)]
+struct ResponseRoot<T> {
+    data: T,
+}
+
+#[allow(non_snake_case)]
+#[derive(Deserialize, Debug)]
+struct AppMetadataResponseData {
+    metadata: ServiceAppMetadata,
+    tags: Vec<TagRecord>,
+}
+
+#[allow(non_snake_case)]
```

Comment:
Another way is to use the regular rust naming convention and change the serialization with `#[serde(rename_all = "camelCase")]`